SignaturePad no longer usable in typescript starting with 5.0.5

In our (react, typescript) project, upgrading to 5.0.5 (and 5.0.6) fails with the following import issues:

```
npx tsc
src/app.ts:3:28 - error TS2307: Cannot find module 'signature_pad/dist/types/point' or its corresponding type declarations.

3 import { BasicPoint } from 'signature_pad/dist/types/point';
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in src/app.ts:3
```

In 5.0.4 this works fine. Issues startet with 5.0.5 and are still there with 5.0.6. Is it possible to tune the exports so that importing `BasicPoint` is possible from our typescript project? Or is the issue on our end? Any help is appreciated.

Minimal viable reproducer is here:
https://github.com/simschla/signature-pad-import-issues-reproducer

(never mind the code: it does not make sense - it just shows the issue we have in our production code).

--- a/src/signature_pad.ts
+++ b/src/signature_pad.ts
@@ -14,6 +14,8 @@ import { BasicPoint, Point } from './point';
 import { SignatureEventTarget } from './signature_event_target';
 import { throttle } from './throttle';
 
+export { BasicPoint } from './point';
+
 export interface SignatureEvent {
   event: MouseEvent | TouchEvent | PointerEvent;
   type: string;

